### ISSUE TITLE
Accessibility Updates

### ISSUE THREAD

**Issue by jnschrag** [CSIS-iLab/ocean#200]
## General
- [x] When we have an external link that has `target="_blank"`, we should add `rel="noopener"` **A**
  - Ex: csis.org, Privacy Policy link, all of the further reading on the Spotlight, the social share links

## Spotlight
- [x] Remove `role="figure"` from `<figure>` tags **A**
- [x] Buttons should have an accessible name attribute, specifically the video play button and the restart button on the timeline **L**
- [x] Add a name attribute to the Email link in the social share. The text can be "Email link to [NAME OF PAGE]" **A**
- [x] Add a name attribute to the print link in the social share. The text can be "Print [NAME OF PAGE]" **A**

## Archive Pages
- [x] Remove `role="content"` from `<article class="archive">` tag **A**
- [x] Remove `role="archive__footer"` from `<article class="archive__footer">` **A**
- [x] On the search page, a lot of the elements are missing proper labels. The text like "Sort By" was done with pseudo elements instead. At the very least, there should be a hidden label for screen readers. This should apply to the search bar and all dropdowns. **L**
